src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public static Ball getRandomBall() {
+
+        Color randomColor = ColorSupplier.getRandomColor();
+        Random random = new Random();
```

Comment:
new Random object is created each time the method is called. Let's move it to the class fields

Reply:
not fixed